src/output.c

```diff
@@ -15,7 +15,27 @@ void show(num_t result)
 {
 	if (haderror()) return;
 	if (!isclean) fputs("= ", stdout);
-	printf("%g\n", result);
+	real_t real = creal(result);
+	real_t imag = cimag(result);
+	if (imag == 0) {
+		printf("%g\n", real);
+	} else if (real == 0) {
+		if (imag == 1) {
```

Comment:
turn inside-out (if else-if on the value of `imag` on the outside, if-else of the value of `real` on the inside).